recipes/trisbm/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
This file can be removed as it will found from the source tarball (you still need the license_file in the meta.yaml)